Common/TableProducer/PID/pidTOF.cxx

```diff
@@ -26,7 +26,7 @@
 // O2Physics includes
 #include "TableHelper.h"
 #include "pidTOFBase.h"
-#include "DPG/Tasks/qaPIDTOF.h"
+#include "DPG/Tasks/AOTTrack/PID/qaPIDTOF.h"
```

Comment:
Should header files which are used in TableProducer from Tasks be in Common instead?

Reply:
This is just to have the possibility to have `--add-qa` option to the PID producer tasks. The header falls under DPG as it is a task for QA, we can remove it if needed.

Reply:
I see. OK for me